### PR TITLE
fix: add missing monitor attributes

### LINKED ISSUE #90: Cannot convert new fields
**Describe the bug**
The plugin cannot convert these fields in DD monitor JSON:
- `restriction_query`
- `on_missing_data`
- `restricted_roles`

**To Reproduce**
```json
{
	"id": 10000000000,
	"name": "Console: A high error rate",
	"type": "rum alert",
	"query": "rum(\"-env:dev @error.source:(console OR source) @type:error\").rollup(\"count\").last(\"15m\") > 3",
	"message": "A high error rate",
	"tags": [
		"env:prod"
	],
	"options": {
		"thresholds": {
			"critical": 3
		},
		"enable_logs_sample": false,
		"notify_audit": false,
		"restriction_query": null,
		"on_missing_data": "default",
		"include_tags": false,
		"new_host_delay": 300,
		"groupby_simple_monitor": false,
		"silenced": {}
	},
	"priority": 2,
	"restricted_roles": [
		"ROLE_ID"
	]
}
```

**Screenshots**
image

image

image

### PR DESCRIPTION
**Pull request by vigohe**
# Why?

renotify_statuses, restricted_roles and on_missing_data attributes are missing at the monitor converter.

Fix #90 

# How?
Add the missing attributes to monitor-converter.js

## Before
can't convert key 'renotify_statuses'
can't convert key 'restricted_roles'
can't convert key 'on_missing_data'

### CODE REVIEW

File: app/monitor-converter.js [hunk 1]
```diff
@@ -32,6 +33,8 @@ const OPTIONS = {
   groupby_simple_monitor: (v) => assignmentString("groupby_simple_monitor", v),
   silenced: (_) => "", // 2.23.0 deprecated
   new_group_delay: (v) => assignmentString("new_group_delay", v),
+  renotify_statuses: (v) => assignmentString("renotify_statuses", v),
+  on_missing_data: (v) => assignmentString("on_missing_data", v),
```
Comment (asfaltboy) [inline app/monitor-converter.js:37]:
```suggestion
  on_missing_data: (v) => assignmentString("on_missing_data", v),
  restriction_query: (v) => assignmentString("restriction_query", v),
```

File: examples/monitor.json [hunk 2]
```diff
@@ -20,14 +20,17 @@
     "escalation_message": "",
     "no_data_timeframe": 10,
     "include_tags": true,
+    "on_missing_data": "default",
```
Comment (asfaltboy) [inline examples/monitor.json:23]:
```suggestion
    "on_missing_data": "default",
    "restriction_query": null,
```

**Review by asfaltboy — APPROVED** [review]
I'm not super familiar with the logic or the schema, but my export included the `restriction_query` field with the value `null`. I've added this field in the suggestions below. What do you think @vigohe ?

**Review by laurmurclar — APPROVED** [review]
Nice! Thank you so much for contributing this. I'll merge it now and then cut a release. It might take a few hours to a few days to get approved by the Chrome & Firefox teams, but you should see it working soon

**Pull request merged**